src/wp-admin/nav-menus.php

```diff
@@ -582,22 +607,38 @@ function wp_nav_menu_max_depth( $classes ) {
 		$focus = $locations_screen ? array( 'section' => 'menu_locations' ) : array( 'panel' => 'nav_menus' );
 		printf(
 			' <a class="page-title-action hide-if-no-customize" href="%1$s">%2$s</a>',
-			esc_url( add_query_arg( array(
-				array( 'autofocus' => $focus ),
-				'return' => urlencode( remove_query_arg( wp_removable_query_args(), wp_unslash( $_SERVER['REQUEST_URI'] ) ) ),
-			), admin_url( 'customize.php' ) ) ),
+			esc_url(
+				add_query_arg(
+					array(
+						array( 'autofocus' => $focus ),
+						'return' => urlencode( remove_query_arg( wp_removable_query_args(), wp_unslash( $_SERVER['REQUEST_URI'] ) ) ),
+					),
+					admin_url( 'customize.php' )
+				)
+			),
 			__( 'Manage with Live Preview' )
 		);
 	endif;
+
+	$nav_tab_active_class = '';
+	if ( ! isset( $_GET['action'] ) || isset( $_GET['action'] ) && 'locations' != $_GET['action'] ) {
+		$nav_tab_active_class = ' nav-tab-active';
+	}
 	?>
 
 	<hr class="wp-header-end">
 
 	<h2 class="nav-tab-wrapper wp-clearfix">
-		<a href="<?php echo admin_url( 'nav-menus.php' ); ?>" class="nav-tab<?php if ( ! isset( $_GET['action'] ) || isset( $_GET['action'] ) && 'locations' != $_GET['action'] ) echo ' nav-tab-active'; ?>"><?php esc_html_e( 'Edit Menus' ); ?></a>
+		<a href="<?php echo admin_url( 'nav-menus.php' ); ?>" class="nav-tab<?php echo $nav_tab_active_class; ?>"><?php esc_html_e( 'Edit Menus' ); ?></a>
 		<?php if ( $num_locations && $menu_count ) : ?>
-			<a href="<?php echo esc_url( add_query_arg( array( 'action' => 'locations' ), admin_url( 'nav-menus.php' ) ) ); ?>" class="nav-tab<?php if ( $locations_screen ) echo ' nav-tab-active'; ?>"><?php esc_html_e( 'Manage Locations' ); ?></a>
-		<?php
+			<a href="<?php echo esc_url( add_query_arg( array( 'action' => 'locations' ), admin_url( 'nav-menus.php' ) ) ); ?>" class="nav-tab
+				<?php
```

Comment:
newline in class attribute can affect attribute value as CSS selector

Reply:
PR updated.